packages/vuetify/src/effects/elevation/__tests__/elevation.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Setup
+import { defineComponent } from 'vue'
+
+// Mixins
+import {
+  ElevationProps,
+  useElevation,
+} from '../'
+
+// Utilities
+import { mount } from '@vue/test-utils'
+
+describe('elevation.ts', () => {
+  /* tslint:disable-next-line */
+  const MockComponent = defineComponent({
```

Comment:
Unit test should test the effect itself in the first place, not its usage in component/setup

Reply:
Unit testing doesn't even work right now.

Reply:
Unit tests don't work in v3, no effort really put into them.